### PR TITLE
[Tokens] Update css-library to 0.16.0 to get feedback tokens

### PR DESCRIPTION
**Pull request by narin**
## Description of Change

Updated css-library to 0.16.0 to grab 6 color tokens we had them add:

**Primitive**
- uswds-system-color-green-cool-vivid-30
- uswds-system-color-gold-vivid-40

**Semantic**
- vads-color-feedback-foregound-success-on-dark: uswds-system-color-green-cool-vivid-30
- vads-color-feedback-foregound-warning-on-light: uswds-system-color-gold-vivid-40
- vads-color-feedback-foregound-success-on-light: uswds-system-color-green-cool-vivid-50
- vads-color-feedback-foregound-warning-on-dark: uswds-system-color-gold-vivid-30

Also, VADS web team added their own font tokens in one of the recent versions, some of which conflict with our font tokens. I've added a filter to ignore web font tokens since there is no crossover for fonts between web and mobile.

#### Testing Packages

- 0.20.1-alpha.0

## Testing

- [x] Tested on iOS 
- [x] Tested on Android 
- [x] Tested on Web

## PR Checklist
Code reviewer validation:
- General
	- [ ] PR is linked to ticket(s)
	- [x] PR has `changelog` label applied if it's to be included in the changelog
	- [ ] Acceptance criteria: 
		- All satisfied _or_
		- Documented reason for not being performed _or_
		- Split to separate ticket and ticket is linked by relevant AC(s)
	- [x] Above PR sections adequately filled out
	- [x] If any breaking changes, in accordance with the [pre-1.0.0 versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library#versioning-policy): a CU ticket has been created for the VA Mobile App detailing necessary adjustments with the package version that will be published by this ticket
- Code
	- [x] Tests are included if appropriate (or split to separate ticket)
	- [x] New functions have proper TSDoc annotations

## Publish

If changes warrant a new version [per the versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library/blob/main/documentation/versioning.md) and the PR is approved and ready to merge:
- [ ] Merge `main` into branch
- [ ] Merge branch to `main`
- [ ] Verify that [Check Component Integrations](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/check-component-integrations.yml) workflow ran successfully
- [ ] [Publish new version](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/publish.yml) after merging

### CODE REVIEW

File: packages/tokens/build.js [hunk 1]
```diff
@@ -43,11 +43,16 @@ const stripModeReducer = (result, token) => {
...
 
-/** Filter function to return tokens of category 'font', type from filter, and npm true */
+/** Filter function to return tokens of category 'font', type from filter, and npm true. Exclude web font tokens */
 const filterFont = (token, fontType) => {
   const { category, type, npm } = token.attributes
 
-  return category === 'font' && type === fontType && npm === true
+  return (
+    category !== 'vads-font' &&
+    category === 'font' &&
```
Comment (TimRoe) [inline packages/tokens/build.js:52]: Isn't this redundant? If the category is equal to `font`, it's definitely not `vads-font`.

If we did want to include it to somewhat document in the code, maybe instead do it as a separate return of: `if (category === 'vads-font') return false` and leave the original return relating to our area of interest as it was.

File: packages/tokens/build.js [hunk 2]
```diff
@@ -115,6 +120,12 @@ StyleDictionary.registerFilter({
   matcher: (token) => token.attributes.category.includes('spacing'),
 })
 
+/** Remove VADS web font tokens */
+StyleDictionary.registerFilter({
+  name: 'filter/fonts/mobile-fonts',
+  matcher: (token) => token.attributes.category !== 'vads-font',
```
Comment (TimRoe) [inline packages/tokens/build.js:126]: Not sure if there's another way without digging at all, but just noting it feels weird to have a general filter that needs to be applied to all files that didn't previously have a filter to get rid of the `vads-font`s.

**Review by TimRoe — APPROVED** [review]
Approving since nothing necessarily needs to change, but have some comments/suggestions. 

Also noting it's not linked to a ticket (is it supposed to be just to the same ticket as VADS?).

File: packages/tokens/build.js (hunk 2 above)
Comment (narin) [reply packages/tokens/build.js:120]: There doesn't seem to be a way to apply filters to an entire platform or transform group, but style-dictionary 4.0 introduces preprocessors which might be able to clean this up. Created https://github.com/department-of-veterans-affairs/va-mobile-library/issues/604 to follow up on it after we update to 4.0.

**Pull request merged**